pom.xml

```diff
@@ -116,7 +116,7 @@
         <skipPerformanceTests>true</skipPerformanceTests>
         <skipArchetypeTests>true</skipArchetypeTests>
 
-        <skipDistributions>true</skipDistributions>
+        <skipDistributions>false</skipDistributions>
```

Comment:
why skipDistributions false? This should be default in mule 3 and enabled by a command line property when needed. I don't think we should change the behavior as this could disrupt several pipelines.

Reply:
I was testing locallly, will revert